src/java.desktop/share/classes/javax/swing/ImageIcon.java

```diff
@@ -61,6 +61,11 @@
  * from Images. Images that are created from a URL, filename or byte array
  * are preloaded using MediaTracker to monitor the loaded state
  * of the image.
+ * If the image source parameter to a constructor or method is non-null,
```

Comment:
```suggestion
 * of the image.
 *
 * <p>
 * If the image source parameter to a constructor or method is non-null,
```
I suggest putting the discussion on non-null parameters in its own paragraph for ease of reading the javadoc.

Reply:
ok